src/pages/Models/ModelDetails/ModelDetails.test.js

```diff
@@ -1,11 +1,35 @@
+import { Provider } from "react-redux";
+import { mount } from "enzyme";
+import configureStore from "redux-mock-store";
 import React from "react";
-import { shallow } from "enzyme";
+
+import dataDump from "testing/complete-redux-store-dump";
 
 import ModelDetail from "./ModelDetails";
 
+const mockStore = configureStore([]);
+
 describe("ModelDetail Container", () => {
-  it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<ModelDetail />);
+  it("renders with a login message when not logged in", () => {
+    const store = mockStore({
+      root: {},
+      juju: dataDump.juju
+    });
+    const wrapper = mount(
+      <Provider store={store}>
+        <ModelDetail />
+      </Provider>
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
These snapshots are probably not that useful when using `mount()`. Probably better to check that a few things exist.

At the very least you should do something like `expect(wrapper.find("ModelDetail")).toMatchSnapshot();` so that you're not including the provider in the snapshot.